mitosheet/mitosheet/tests/public/v3/sheet_functions/number_functions/test_sumproduct.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+
+import pytest
+import pandas as pd
+import numpy as np
+from mitosheet.public.v3.rolling_range import RollingRange
+
+from mitosheet.public.v3.sheet_functions.number_functions import SUMPRODUCT
+
+
+SUM_PRODUCT_VALID_TETS = [
+    # Series
+    ([pd.Series([1,2,3]), pd.Series([1,1,1])], 6),
+    ([pd.Series([1,2,3]), pd.Series([1,2,3])], 14),
+    ([pd.Series([1,2,3]), pd.Series([1,2,3.0])], 14.0),
+    ([pd.Series([1,2,3]), pd.Series([1,2,3]), pd.Series([1,2,3])], 1 + 8 + 27),
+    
+    # Dataframes and series
+    ([pd.DataFrame({'a': [1, 1, 1], 'b': [2, 2, 2]}), pd.DataFrame({'a': [1, 1, 1], 'b': [2, 2, 2]}), pd.DataFrame({'a': [1, 1, 1], 'b': [2, 2, 2]})], 3 + 8 * 3),
+]
+
+SUM_INVALID_CAST_TESTS = [
```

Comment:
Can we add a test that makes sure rolling ranges does not work in the way we expect?